src/templates/_layouts/components/global-sidebar.twig

```diff
@@ -131,12 +131,13 @@
             </div>
 
             {% if currentUser.admin and devMode %}
-                {% set devModeText = 'Craft CMS is running in Dev Mode.'|t('app') %}
+                {% set devModeText = '<span lang="en">Craft CMS</span> is running in Dev Mode.'|t('app') %}
```

Comment:
@brandonkelly we should update all instances of "Craft CMS" and other brand names to include a wrapper with `lang="en"`. Is changing the original translation string the best way of going about this, or should a new one be added?

Reply:
@brandonkelly sounds good! The rest are lower priority since only a couple were in the representative sample we used for the audit, so I’ll update those separately.